scss/forms/_contribute.scss

```diff
@@ -0,0 +1,7 @@
+.___outside-namespace {
+  .CRM_Contribute_Form_Contribution {
+    ~ .prospect__case-info {
+      top: 110px;
```

Comment:
Is it possible to assign a variable to this value. Why 110px?

Reply:
Its a magic number, so that the `prospect__case-info` does not overlap(as shown on screenshot). Again, it wont be reused, so I dont see a point to create a variable.